.kitchen.cloud.yml

```diff
@@ -17,17 +17,23 @@ provisioner:
   name: chef_solo
   data_bags_path: test/integration/default/data_bags
   encrypted_data_bag_secret_key_path: test/integration/default/encrypted_data_bag_secret
-  attributes:
 
 platforms:
-  - name: centos-6.5
+  - name: centos-6.6
     driver_plugin: openstack
     driver_config:
       username: centos
-      image_ref: "CentOS 6.5"
+      image_ref: "CentOS 6.6"
 
 suites:
   - name: default
     run_list:
       - recipe[osl-jenkins::default]
     attributes:
+  - name: cookbook-uploader
+    run_list:
+      - recipe[java]
+      - recipe[jenkins::master]
+      - recipe[osl-jenkins::cookbook-uploader]
+    data_bags_path: ../../data_bags/
+    encrypted_data_bag_secret_key_path: ../../../../.chef/encrypted_data_bag_secret
```

Comment:
It's best to create a new `data_bags` and `encrypted_data_bag_secret` for testing purposes under `test/`